pyproject.toml

```diff
@@ -208,6 +208,7 @@ transformations = [
   "statsmodels<0.15,>=0.12.1",
   'stumpy<1.13,>=1.5.1; python_version < "3.12"',
   'tsfresh<0.21,>=0.17; python_version < "3.12"',
+  "tsbootstrap@git+https://github.com/astrogilda/tsbootstrap.git",
```

Comment:
didn't know you could do that! Thanks for the learning moment.

Reply:
You are welcome. I will change it, if tsbootstrap 0.0.1 is released.

Reply:
you mean 0.1.0, right?

Reply:
Yes